src/components/Repos/NotesForm.js

```diff
@@ -0,0 +1,39 @@
+import { useParams, useNavigate } from "react-router-dom"
+import { useState } from 'react'
+
+function NotesForm () {
+  const {username, reponame} = useParams()
+  const [note, setNote] = useState({
+    contents: '',
+    repo: reponame,
+    repoOwner: username
+  })
+  const navigate = useNavigate()
+
+  const handleSubmit = async (e) => {
+    e.preventDefault()
+    await fetch(`http://localhost:4000/notes`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({...note, timestamp: new Date()})
```

Comment:
Where is the ID being added in to your data store objects? Handle change is changing the content value, and this is adding the timestamp when it is posted, but I may have missed where the ID is being generated and how it is incrementing to ensure it is unique for each note being added?

Reply:
I believe that when you create the object the ID is automatically generated.